src/main/blackbox/blackbox.h

```diff
@@ -61,7 +61,7 @@ typedef enum FlightLogEvent {
 typedef struct blackboxConfig_s {
     uint8_t sample_rate; // sample rate
     uint8_t device;
-    uint8_t record_acc;
+    uint32_t fields_mask; // 1 to not log that field, 0 to log that field
```

Comment:
This should really be changed to `fields_disabled_mask` for clarity - then it won't need a comment. (As a guideline when coding, whenever you are about to add a comment, ask yourself 'could I change this so it does not need a comment'.)